transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/BiosampleTransformations.scala

```diff
@@ -38,7 +61,9 @@ object BiosampleTransformations {
       replicationType = "repType",
       treatments = biosampleInput.read[Array[String]]("treatments"),
       wasPerturbed = biosampleInput.read[Boolean]("perturbed"),
-      submittedBy = biosampleInput.read[String]("submitted_by")
+      submittedBy = biosampleInput.read[String]("submitted_by"),
+      productId = product_ids.headOption,
+      lotId = lot_ids.headOption
```

Comment:
Should we do `headOption` here (effectively choosing randomly) if there are > 1 IDs? Or should we leave it as `None` to mark an error state?

Reply:
I like that suggestion. I'll change it to store None when there's an error.

Reply:
I just made the change. Let me know if it's what you had in mind.